public/app/features/alerting/unified/components/admin/AlertmanagerConfig.tsx

```diff
@@ -84,80 +97,43 @@ export default function AlertmanagerConfig(): JSX.Element {
         dataSources={alertManagers}
       />
       {loadingError && !loading && (
-        <Alert severity="error" title="Error loading Alertmanager configuration">
-          {loadingError.message || 'Unknown error.'}
-        </Alert>
+        <>
+          <Alert
+            severity="error"
+            title="Your Alertmanager configuration is incorrect. These are the details of the error:"
+          >
+            {loadingError.message || 'Unknown error.'}
+          </Alert>
+
+          {alertManagerSourceName === GRAFANA_RULES_SOURCE_NAME && (
+            <AlertmanagerConfigSelector
```

Comment:
nit: can we move the "Reset to selected configuration" button next to the drop-down so the user doesn't have to scroll down?

Reply:
Done! 
![image](https://user-images.githubusercontent.com/6271380/229505900-e59c6833-e6cf-47aa-83f9-3f8be8990960.png)